src/Features/LanguageServer/Protocol/Features/Options/LanguageServerProjectSystemOptionsStorage.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace
+{
+    internal static class LanguageServerProjectSystemOptionsStorage
+    {
+        private static readonly OptionGroup s_optionGroup = new(name: "project_loading", description: "");
```

Comment:
I couldn't bring myself to use the term "project system" in a user-facing option. Unless maybe just call it "projects", so dotnet.projects.binaryLogPath?

Reply:
I think I prefer projects